config/manifests/bases/habana-ai-operator.clusterserviceversion.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     containerImage: quay.io/fabiendupont/habana-ai-operator
     description: Manages Habana Labs accelerators within a Kubernetes cluster
     olm.skipRange: '>=0.0.0 <99.0.0'
+    operatorframework.io/cluster-monitoring: true
```

Comment:
According to this [doc](https://github.com/openshift/enhancements/blob/master/enhancements/olm/olm-managed-operator-metrics.md#fulfilling-namespace-and-rbac-requirements), this should be the only change we need to enable cluster monitoring for the operator.

But I am not sure if this is actually effective for namespaces that don't start with the prefix `openshift-`.

When I follow this [doc](https://docs.openshift.com/container-platform/4.11/operators/operator_sdk/osdk-monitoring-prometheus.html#osdk-monitoring-custom-metrics_osdk-monitoring-prometheus), which actually guides you to implement the required changes manually, then I can see the operator metrics in the `openshif-monitoring` prometheus instance.